services/slam/builtin/builtin.go

```diff
@@ -457,6 +479,44 @@ func (slamSvc *builtIn) GetMap(
 	if err != nil {
 		return "", nil, nil, err
 	}
+	reqPCMap := &pb.GetPointCloudMapRequest{
+		Name: name,
+	}
+
+	var imData image.Image
+	var vObj *vision.Object
+
+	// TODO: Once RSDK-1053 (https://viam.atlassian.net/browse/RSDK-1066) is complete the original code that extracts
+	// the map will be removed and GetMap will be changed to GetPointCloudMap
+	if slamSvc.dev {
+		slamSvc.logger.Debug("IN DEV MODE (map request)")
+
+		if mimeType != rdkutils.MimeTypePCD {
```

Comment:
[opt] move this check to the `if slamSvc.dev` line, so the jpeg endpoint still works in dev mode. Whoever adds the do functionality can remove the check in a later PR

Reply:
+1